docs/semgrep-appsec-platform/jira.md

```diff
@@ -127,6 +128,14 @@ If you opt for a drop-down or a checkbox issue type field, verify that:
 
 After setting up your Jira integration, you're now ready to create Jira tickets. Jira tickets can be created from findings in Semgrep Code, Supply Chain, and Secrets.
 
+### Component Mappings
+
+If you've created a custom field mapping for a component field type, be aware that if you choose to create tickets in a project other than your default project as configured in your integration settings, you'll need to ensure that project has access to the component you selected for your mapping.
+
+If your default project uses [**Jira Components**](https://support.atlassian.com/jira-software-cloud/docs/what-are-jira-components/) and you create a component field mapping in your integration settings, **you'll be unable to create tickets in another project** because Jira components are always project-scoped.
```

Comment:
```suggestion
If your default project uses [**Jira Components**](https://support.atlassian.com/jira-software-cloud/docs/what-are-jira-components/) and you create a component field mapping in your integration settings, **you cannot create tickets in another project** because Jira components are always project-scoped.
```